app/views/hyrax/my/_collection_action_menu.html.erb

```diff
@@ -30,14 +31,18 @@
         <% end %>
       </li>
     <% end %>
-    <%# TODO: Only managers and depositors can add collections to collections. %>
-    <li role="menuitem" tabindex="-1">
-      <%= link_to '#',
-        class: 'itemicon add-collection-to-collection',
-        title: t("hyrax.dashboard.my.action.add_collection_to_collection"),
-        data: { nestable: collection_presenter.collection_type_is_nestable? } do %>
-        <%= t("hyrax.dashboard.my.action.add_collection_to_collection") %>
-      <% end %>
-    </li>
+    <%# if no collecton types are nestable, then do not allow mapping option to show up. %>
+<% byebug %>
+    <% if Collection.all.map {|c| c.nestable? }.include?(true) %>
```

Comment:
Is nestable? not available from solr doc?  I had to have to get the collection from Fedora just to determine if it is nestable.